lib/campaign/CampaignState.cpp

```diff
@@ -107,6 +108,11 @@ ImagePath CampaignRegions::getConqueredName(CampaignScenarioID which, int color)
 	return getNameFor(which, color, "Co");
 }
 
+int CampaignRegions::getRegionCount() const
```

Comment:
```suggestion
int CampaignRegions::getRegionsCount() const
```

Note: regions count not equivalent to scenarios count